Interfaces/scripts/dirac-wms-job-submit.py

```diff
@@ -11,11 +11,15 @@
 import DIRAC
 from DIRAC.Core.Base import Script
 
+import os
+
 Script.setUsageMessage( '\n'.join( [ __doc__.split( '\n' )[1],
                                      'Usage:',
                                      '  %s [option|cfgfile] ... JDL ...' % Script.scriptName,
                                      'Arguments:',
                                      '  JDL:      Path to JDL file' ] ) )
+
+Script.registerSwitch( "f:", "File=", "Writes job ids to file <value>" )
```

Comment:
This was written to be compatible with the dirac-wms-job-status -f option, which also doesn't use JobRepository (presumably so that a human could also easily write such a file by hand).